### ISSUE TITLE
class ModelBaseEKFFlexEstimatorIMU has no member named setWithUnmodeledMeasurements

### ISSUE THREAD

**Issue by andreadelprete** [stack-of-tasks/sot-state-observation#10]
Hi, I just downloaded the latest versions of state-observation and sot-state-observation and when I try to compile the latter I get this error:
```
/home/adelpret/devel/openrobots/src/sot-state-observation/include/sot-state-observation/dg-imu-model-base-flex-estimation.hh:237:50: error: ‘class stateObservation::flexibilityEstimation::ModelBaseEKFFlexEstimatorIMU’ has no member named ‘setWithUnmodeledMeasurements’
                     if (config(0)==1) estimator_.setWithUnmodeledMeasurements(true);
```
Any idea what the problem could be?
@mehdi-benallegue @amifsud

**Comment by mehdi-benallegue** [comment]
Dear Anrea,

Can you precise me the repo you downloaded it from ?

Mehdi-

2017-07-10 16:47 GMT+09:00 Andrea Del Prete <notifications@github.com>:

**Comment by mehdi-benallegue** [comment]
I think I know the origin of the problem. sot-state observation  is managed by @amifsud, and state-observation is managed by me. @amifsud  did not yet fully merge our code together. So you could try to pull the version from his repo 
https://github.com/amifsud/sot-state-observation

**Comment by andreadelprete** [comment]
I was using the master branch of the sot repos. I will try with the one you suggested, thanks!

**Comment by amifsud** [comment]
Yes, I saw the problem but I didn't fix it yet. You can checkout commit 745c28967585 on sot-state-observation package and commit 596d369027c73780 on state-observation package.

**Comment by andreadelprete** [comment]
I managed to compile using the branch topic/timeComputation. Thanks @amifsud !
